packages/ovm/src/contracts/ExecutionManager.sol

```diff
@@ -118,52 +117,37 @@ contract ExecutionManager is FullStateManager {
      *       [callBytes (bytes (variable length))]
      * returndata: [variable-length bytes returned from call]
      */
-    function executeCall() external {
+    function executeTransactionRaw() external {
         uint _timestamp;
         uint _queueOrigin;
-        uint callSize;
-        bytes memory callBytes;
-        bytes32 methodId = ovmCallMethodId;
+        uint _callSize;
+        bytes memory _callBytes;
+        address _ovmEntrypoint;
```

Comment:
I had been going with the convention of `_expectedCalldataParameters` and `regularVariables` to go along with regular solidity convention.  `callBytes` is certainly a hybrid. Just a thought -- I'm open to whatever though.